src/mesh/mesh_tetgen_interface.C

```diff
@@ -39,8 +39,22 @@ TetGenMeshInterface::TetGenMeshInterface (UnstructuredMesh & mesh) :
   _mesh(mesh),
   _serializer(_mesh)
 {
+  _switches="Q";
```

Comment:
Thanks for fixing this, but it should just be initialized in the initialization list (two lines up) like the other members.